### PR TITLE
Convert email to support

### PR DESCRIPTION
**Pull request by andy-t-wang**
Update support to accommodate both email and support.

I didn't update the field yet, pending designs

### CODE REVIEW

File: web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx [hunk 1]
```diff
@@ -30,18 +30,17 @@ import {
...
   is_whitelist_disabled: yup.boolean(),
-  support_email: yup.string().when("app_mode", {
+  support_link: yup.string().when("app_mode", {
     is: true,
     then: (schema) =>
-      schema
-        .email("Must be a valid email address")
-        .required("This field is required"),
-    otherwise: (schema) => schema.notRequired(),
+      schema.matches(urlRegex, "URL must start with https:// or mailto::"),
```
Comment (igorosip0v) [inline web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx:43]:
```suggestion
      schema.matches(urlRegex, "URL must start with https:// or mailto:"),
```

File: web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx [hunk 2]
```diff
@@ -428,19 +427,19 @@ export const MiniAppForm = (props: LinksFormProps) => {
...
         </div>
+        {/* Pending designs change this to a switcher */}
         <Input
-          label="Support Email"
+          label="Support Link"
           disabled={!isEditable || !isEnoughPermissions || !appMode}
-          placeholder="address@example.com"
-          register={register("support_email")}
-          errors={errors.support_email}
+          placeholder="mailto::address@example.com"
```
Comment (igorosip0v) [inline web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx:440]: @andy-t-wang Why are you using two colon symbols? Isn't the correct format with the one colon?

Comment (igorosip0v) [reply web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx:440]:
```suggestion
          placeholder="mailto:address@example.com"
```

File: web/tests/api/v2/minikit-app-metadata.test.ts [hunk 3]
```diff
@@ -83,7 +83,7 @@ const validAppMetadataResponse = [
     integration_url: "https://github.com",
     app_website_url: "",
     source_code_url: "",
-    support_email: "test@test.com",
+    support_link: "test@test.com",
```
Comment (igorosip0v) [inline web/tests/api/v2/minikit-app-metadata.test.ts:86]:
```suggestion
    support_link: "mailto:test@test.com",
```

Comment (igorosip0v) [reply web/tests/api/v2/minikit-app-metadata.test.ts:86]: Also, check other code lines, it seems missing there too.

**Review by igorosip0v — APPROVED** [review]
@andy-t-wang Please update actions in ci.yaml to v4 to fix checks

File: web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx (hunk 2 above)
Comment (andy-t-wang) [reply web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx:440]: ah yeah good catch

**Pull request merged**